pkgs/applications/radio/limesuite/default.nix

```diff
@@ -2,7 +2,7 @@
 , sqlite, wxGTK32, libusb1, soapysdr
 , mesa_glu, libX11, gnuplot, fltk
 , GLUT
-, withGui ? true
+, withGui ? !stdenv.isDarwin
```

Comment:
a short comment why would be very helpful

Reply:
comment added